lib/sentry/transport.ex

```diff
@@ -40,6 +40,11 @@ defmodule Sentry.Transport do
     result
   end
 
+  def record_discarded_event(reason, category) do
+    ClientReport.add_discarded_event({reason, DataCategory.data_category_mapping(category)})
```

Comment:
Why go through the extra `data_category_mapping/1` step here rather than just do the mapping in `ClientReport`, without introducing a new `Sentry.DataCategory` module?

Reply:
I believe we resolved this in our pairing. Updates made so lemme know thoughts